specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/TasksCommon.json

```diff
@@ -545,7 +889,7 @@
     },
     "StartMigrationScenarioServerRoleResult": {
       "type": "object",
-      "description": "Server role migration result",
+      "description": "",
```

Comment:
No description? Then we can get rid of it only?

Reply:
Should not be removed.  Seems to be missing in 2017-11-15-privatepreview so I'll add it in our private repo as well.